pages/index.js

```diff
@@ -1,20 +1,21 @@
 import styled from "styled-components";
 import DonationListItem from "@/components/DonationListItem";
-import donations from "../db.json";
+import Link from "next/link";
 
-export default function HomePage() {
+export default function HomePage({ donations }) {
   return (
     <>
+      <Link href={"/myDonations"}>MyDonations</Link>
       <h2> Spenden in deiner Nähe: </h2>
-      <StyledList>
+      <StyledCard>
```

Comment:
your styledCard is a List in reality, each ul is a List, each item inside the list is a Card.

Reply:
changed it back to <StyledList>